packages/adapter/src/request-building/authentication.ts

```diff
@@ -75,24 +75,27 @@ export function buildParameters(options: CachedBuildRequestOptions): Authenticat
   };
 
   const { securitySchemes } = options;
-  // Security schemes originate from 'security.json' and contain all of the authentication details
+  // Security schemes originate from 'config.json' and contain all of the authentication details
```

Comment:
I think changing the name of this will help a lot with the confusion (like I mentioned below). These objects are not actually `securitySchemes`